src/extension/wmts-multi-dimensional/src/test/java/org/geoserver/gwc/wmts/VectorElevationDimensionTest.java

```diff
@@ -50,6 +51,38 @@ public void testDisabledDimension() throws Exception {
                 DimensionsUtils.extractDimensions(wms, getLayerInfo(), ALL_DOMAINS).size(), is(0));
     }
 
+    @Test
+    public void testFixedValueRangeSingleValueDimension() throws Exception {
+        // enable a elevation dimension
+        DimensionInfo dimensionInfo = new DimensionInfoImpl();
+        dimensionInfo.setFixedValueRange("0.1");
+        dimensionInfo.setEnabled(true);
+        FeatureTypeInfo vectorInfo = getVectorInfo();
+        vectorInfo.getMetadata().put(ResourceInfo.ELEVATION, dimensionInfo);
+        getCatalog().save(vectorInfo);
+        // check that we correctly retrieve the elevation dimension
+        assertThat(
+                DimensionsUtils.extractDimensions(wms, getLayerInfo(), ALL_DOMAINS).size(), is(1));
+        // Testing the fixed value range
+        assertEquals("0.1",dimensionInfo.getFixedValueRange());
+    }
+
+    @Test
+    public void testFixedValueRangeListDimension() throws Exception {
+        // enable a elevation dimension
+        DimensionInfo dimensionInfo = new DimensionInfoImpl();
+        dimensionInfo.setFixedValueRange("0.1,1.0");
+        dimensionInfo.setEnabled(true);
+        FeatureTypeInfo vectorInfo = getVectorInfo();
+        vectorInfo.getMetadata().put(ResourceInfo.ELEVATION, dimensionInfo);
+        getCatalog().save(vectorInfo);
+        // check that we correctly retrieve the elevation dimension
+        assertThat(
+                DimensionsUtils.extractDimensions(wms, getLayerInfo(), ALL_DOMAINS).size(), is(1));
+        // Testing the fixed value range
+        assertEquals("0.1,1.0",dimensionInfo.getFixedValueRange());
```

Comment:
the same as the comment below. This test and the one above seem to simply test getter and setter